erpnext/crm/onboarding_step/social_media_post/social_media_post.json

```diff
@@ -0,0 +1,21 @@
+{
+ "action": "Go to Page",
+ "action_label": "Click to Learn Social Media Campaigns",
+ "creation": "2021-07-21 00:31:44.462792",
+ "description": "# Social Media Campaigns\n\nSocial Media Post allows creating and scheduling the post for your official LinkedIn and Twitter handles. Social Media posts can be also linked with a Campaign, eventually linking to Leads and Opportunities.",
```

Comment:
Doesn't navigate to Social Media Campaign. Check comment for Email Group and apply the same.